docs/UserGuide.adoc

```diff
@@ -30,6 +30,13 @@ Please refer to the <<DeveloperGuide#setting-up, Setting up>> section to learn h
 . Type `java seedu.addressbook.Main`, then Enter to execute
 . Now you can interact with the program through the CLI
 
+== Changes from level 2 Origin
+
+* Newly added feature: "info"
+
+    View some information about the current version of addressbook
+    Format: `info`
```

Comment:
Good that you have updated the UserGuide. But I would suggest keeping the list of features in later part of this doc updated to be consistent.